packages/build-utils/src/fs/glob.ts

```diff
@@ -39,9 +39,30 @@ export default async function glob(
 
   options.symlinks = {};
   options.statCache = {};
+  options.cache = {};
   options.stat = true;
   options.dot = true;
 
+  await vanillaGlob(pattern, options); // populates the caches
```

Comment:
Isn't it pretty inefficient if we call `glob` twice with the same pattern?

Reply:
I don't think so because the second run reuses the caches almost entirely

Reply:
Unfortunately glob still makes lstat calls even with full cache, so the second run is not in-memory only. Not enough score to continue here. The solution is going to be too hacky. Will implement in another place of the infra.